objects/rct2ww/ride/rct2.ww.sputnikr.json

```diff
@@ -58,7 +58,7 @@
             "ru-RU": "Спутник"
         },
         "description": {
-            "en-GB": "Passengers ride face-down in a lying position, suspended in a specially designed car from the single-rail track, swinging freely from side to side around corners",
+            "en-GB": "Suspended satellite-shaped cars in which the passengers ride face-down in a lying position, swinging freely from side to side",
```

Comment:
The Sputnik Cars behave more like the regular Swinging Cars stat-wise, so I would have the description of: "Russian satellite-shaped cars which swing from the rail above"